cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/mazha/ChangesFeedTest.java

```diff
@@ -94,12 +94,44 @@ public void changes_docWithConflicts_conflictsShouldBeReturned() {
     }
 
     @Test
+    public void changes_dbChangesMustSuccessfullyReturnWithSeqInterval() {
+        org.junit.Assume.assumeTrue(ClientTestUtils.isCouchDBV2(client.getRootUri()));
+        Response res1 = ClientTestUtils.createHelloWorldDoc(client);
+        Response res2 = ClientTestUtils.createHelloWorldDoc(client);
+        ClientTestUtils.createHelloWorldDoc(client);
+        Object lastSeq = "0";
+
+        {
+            // Use batch interval and seq_interval of 4
+            ChangesResult changes = client.changes(lastSeq, 4);
+            Map<String, List<String>> changedRevIds = findChangedRevisionIds(changes);
+
+            Assert.assertThat(changedRevIds.size(), is(equalTo(3)));
+            Assert.assertThat(changedRevIds.keySet(), hasItems(res1.getId(), res2.getId()));
+            Assert.assertThat(changedRevIds.get(res1.getId()), hasItem(res1.getRev()));
+            Assert.assertThat(changedRevIds.get(res2.getId()), hasItem(res2.getRev()));
+            // first change always contains a seq
```

Comment:
Doesn't always seem to be the case...
```sh
$ curl https://cloudant.cloudant.com/animaldb/_changes\?seq_interval=3
{"results":[
{"seq":null,"id":"badger","changes":[{"rev":"4-51aa94e4b0ef37271082033bba52b850"}]},
{"seq":"2-g1AAAANTeJydklFqwkAURZ9VKAV_uwL__JBoFPtXt2B1FjAvL0FDHKHot6twC63OJroKN-EaquMNgiKCcfJ1w5Bz75v7JiOi-qQq1BCO5t_xQLjTirL5UrRZBEH78t0y8SLDzy-a-F2pcTqp6soMB68inVD3e89Z3EZ1H0flQYqb1m5TTV5D5pDlT-eOd2SvgATkGLkj70xAY06t3eTd0LmbdiAhh0JvSyNxMjWx3HqGBZ6w2_LKuYP3NICOpkZfSq2vu_rgpB8FusyuCnpD1AhRkF9U7zstsA0wyL9zf747A3YABhkqtSuRjZJ2kB9r9yWyceE9BO_GgU5Pt5Edtw","id":"_design/views101","changes":[{"rev":"12-dfa39d5771438be0671b9aeb9cfaf03f"}]},
{"seq":null,"id":"aardvark","changes":[{"rev":"3-fe45a3e06244adbe7ba145e74e57aba5"}]},
{"seq":null,"id":"elephant","changes":[{"rev":"3-f812228f45b5f4e496250556195372b2"}]},
{"seq":"5-g1AAAAO7eJy1kkFOwkAYhX_BxJi49QTuWJCB2lZWeAWFOcD8_YdgU4fEwJpTcAWFuYSn4BKcQRhfNZEQE2tJWL3JZN57X15bENHVuCl0I5xNXuy9cLedFZOZGDdVqvNzbjs7LfC4YYivtR7m46ZpPOPiQqQbmTT-X8Rh1e3fVWWR5pb3q9xQLcjS5Lkfwq7kPPvitEkSK3sUZ1zRhqLAYB3U5oRpyLn3y5KTvvfsKIk4ErqcObGjJ2flMDOqyETciuchbGvTwLRz5_So9WK_2x2P0kyZE-yGqgGqIG_4XHVpYVvCBvkI4X3PK0nSM2nvNLxbVEEetF4fwYth15BX7ze_3NXdGGkDwb8W4M4_ARwyPao","id":"lemur","changes":[{"rev":"3-552d9dbf91fa914a07756e69b9ceaafa"}]},
{"seq":null,"id":"giraffe","changes":[{"rev":"3-7665c3e66315ff40616cceef62886bd8"}]},
...
```
Is this going to be problematic?

Reply:
You're right, I just tested this on CouchDB 2.0, 2.1, and Cloudant and the first`seq` is `null` if I use an odd number for the `seq_interval` value.

Reply:
Removed first `seq` assert in 8361cbe.